### PR TITLE
[WIP] streamline the whole documentation

### PR DESCRIPTION
**Pull request by OskarStark**
Refs sonata-project/SonataAdminBundle#3310

### CODE REVIEW

**Comment by rande** [comment]
Don't forget to squash commits ;)

File: Resources/doc/reference/advanced_usage.rst [hunk 1]
```diff
@@ -11,30 +11,36 @@ tweak this behavior by ignoring patterns :
...
-        ignore_routes:
-            - sonata_page_cache_esi
-            - sonata_page_cache_ssi
+        # app/config/config.yml
 
-        ignore_uri_patterns:
-            - ^/admin\/     # ignore admin route, ie route containing 'admin'
+        sonata_page:
+            ignore_route_patterns:
+                - (.*)admin(.*)   # ignore admin route, ie route containing 'admin'
```
Comment (javiereguiluz) [inline Resources/doc/reference/advanced_usage.rst]: strictly speaking, `ie` should be spelled `i.e.` (https://en.wikipedia.org/wiki/List_of_Latin_phrases_(I)#id_est)

File: Resources/doc/reference/advanced_usage.rst [hunk 2]
```diff
@@ -11,30 +11,36 @@ tweak this behavior by ignoring patterns :
...
+.. configuration-block::
+
+    .. code-block:: yaml
 
-    sonata_page:
-        page_defaults:
-            homepage: {decorate: false} # disable decoration for homepage route_name
+        sonata_page:
+            page_defaults:
+                homepage: {decorate: false} # disable decoration for homepage route_name
```
Comment (javiereguiluz) [inline Resources/doc/reference/advanced_usage.rst]: `for homepage route_name` -> `for homepage route` ?

File: Resources/doc/reference/installation.rst [hunk 3]
```diff
@@ -58,34 +65,43 @@ Next, be sure to enable the ``Page`` and ``EasyExtends`` bundles in your applica
...
-        slugify_service:   sonata.core.slugify.cocur # old BC value is sonata.core.slugify.native
-        multisite:        host
-        default_template: default # template key from templates section, used as default for
-        templates:
-            default:  { path: 'SonataPageBundle::layout.html.twig',          name: 'default' }
-            2columns: { path: 'SonataPageBundle::2columns_layout.html.twig', name: '2 columns layout' }
+        sonata_page:
+            slugify_service:   sonata.core.slugify.cocur # old BC value is sonata.core.slugify.native
+            multisite:        host
+            default_template: default # template key from templates section, used as default for
```
Comment (javiereguiluz) [inline Resources/doc/reference/installation.rst]: the comment ends like this: `used as default for`  it looks like something is missing at the end.

File: Resources/doc/reference/advanced_usage.rst (hunk 1 above)
Comment (OskarStark) [inline Resources/doc/reference/advanced_usage.rst]: thank you @javiereguiluz 

changed the ``Configuration.php`` here: #541

**Comment by OskarStark** [comment]
@rande squashed them

Thanks for feedback @javiereguiluz

**Pull request merged**